frontend/src/components/molecules/AddTransaction.tsx

```diff
@@ -22,39 +74,50 @@ const AddTransaction: React.FC<{ className?: string }> = props => {
     interval_type,
     interval,
   }: any) => {
-    if (!recurring) {
-      await TransactionActions.doCreateTransaction(
-        {
-          company_id: auth!.selectedCompany!,
-          date,
-          description,
-          money: money * 100,
-          notes,
-          type,
-        },
-        dispatch
-      );
-    } else {
-      await TransactionActions.doCreateRecurringTransaction(
-        {
-          company_id: auth!.selectedCompany!,
-          end_date,
-          interval,
-          interval_type,
-          start_date: date,
-          template: {
+    if (!(description.length > 140) && !(notes.length > 140)) {
+      state.content = 'Transaction added successfully';
+      state.clicker = onButtonClickHandler;
+      snaxDispatch({ type: 'good' });
```

Comment:
Don't know if you want this, as this will trigger before you get a response from the server. If you want to wait for that, move it to after `await ...doCreateTransaction(...`.